### ISSUE TITLE
WSPeerHeaders in swagger.yaml the version number should not have min or max length.

### ISSUE THREAD

**Issue by yairlisk** [LiskHQ/lisk#1744]
### Expected behavior
When starting a new node, it should find new peers with ws on relevant port and connect to them without errors.
### Actual behavior
When starting a node on version with the following attributes in the confg.json
```
"version": "1.0.0-alpha.0",
"minVersion": ">=1.0.0-alpha.0",
```
the node will not connect to any peers.
### Steps to reproduce
start a new node with the following data in config.json
```
"version": "1.0.0-alpha.0",
"minVersion": ">=1.0.0-alpha.0",
```
### Which version(s) does this affect? (Environment, OS, etc...)
1.0.0-alpha.0
Ubuntu Linux
Devnet

**Comment by MaciejBaj** [comment]
There are still places in swagger.yml schema regarding version validation.

**Comment by MaciejBaj** [comment]
Closed by #1746.
